.github/workflows/cicd.yml

```diff
@@ -70,3 +70,43 @@ jobs:
           git add astraconnector_operator.yaml
           git commit -m "Auto updating the operator image tag" -n
           git push --set-upstream origin master
+
+      - name: Build Install Bundle
+        if: github.ref == 'refs/heads/master'
+        run: |
+          export VERSION=$(cat VERSION)
+          make install-bundle
+
+      - name: Get Version
+        if: github.ref == 'refs/heads/master'
+        id: vars
+        run:
+          echo ::set-output name=version::$(cat VERSION)
+
+      - name: Create Release
+        if: github.ref == 'refs/heads/master'
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          TAG_VERSION: "v${{ steps.vars.outputs.version }}"
+        with:
+          tag_name: ${{ env.TAG_VERSION }}
+          release_name: Release ${{ env.TAG_VERSION }}
+          draft: true
```

Comment:
You know, on reflection we wont remember to hit the button.  Lets just go for it